### ISSUE TITLE
Should a block producer topology be included as an example? 

### ISSUE THREAD

**Issue by stevanlohja** [cardano-foundation/developer-portal#1316]
I noticed that there's a relay node topology file example here https://developers.cardano.org/docs/operate-a-stake-pool/cardano-relay-configuration 

However, in the following block produce config there's no topology example. It might be nice to describe an example of a block producer node topology file (unless there is one and I've missed it)

**Comment by rphair** [comment]
I agree this would be helpful for readers that don't see... or are reluctant to assume... that the BP topology is simply a reduction of the relay topology (i.e. same general syntax but certain sections eliminated).  Probably that reduced nature of the BP topology is why a specific example wasn't included for it.

@katomm @sanskys I guess the best place for this would be a file in `docs/operate-a-stake-pool` (`cardano-bp-configuration.md`) at the head of the _Cardano Block Producer Configuration_ section, with an example topology & likely an explanation how to use the slightly different `-bp` default config settings that are distributed with `9.x` versions of the node (cc @CarlosLopezDeLara).

**Comment by sanskys** [comment]
yes, that would be the right place. Since the documentation is more than an year old now, we can collect all suggestions and I can support in updating it.
